src/CodeGen.fs

```diff
@@ -1139,7 +1155,7 @@ let sampleFSharpSystemClientMember query queryName hasVariables useTasks =
                 return Error response.errors
         }}
 
-    {syncMember}
+    member this.{queryName}({args}) = Async.RunSynchronously(this.{queryName}Async{queryArgs})
```

Comment:
@Zaid-Ajaj I suppose that naming must be different, `Async{QueryName}` for `async and `{QueryName}Async` for `task`

Reply:
Any reason why it should be different? I think `{queryName}Async` should be the method name for both `async` and `task`

Reply:
Because by convention all async methods in F# (that use `async` CE) are prefixed with `Async`.
But all async methods in .NET that return `Task` are suffixed with `Async`

Reply:
I still would rather keep the method names the same here, just a personal preference.